index.html

```diff
@@ -32,83 +34,42 @@
         <div class="headline-inner">
           <h1 class="headline-primary">
             <span class="headline-primary-first">Hey there. I’m Mariane</span>
-            <span class="headline-primary-second"
-              >I’m a software developer</span
-            >
+            <span class="headline-primary-second">I’m a software developer</span>
```

Comment:
Hello Shahadat, may I suggest you write your Name instead of Mariane, because this is supposed to be your portfolio.

Reply:
thanks for your review